### PR TITLE
Bug-library-service : We check condition that never gets hit.

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

File: asab/library/service.py [hunk 1]
```diff
@@ -132,13 +132,12 @@ async def _read_disabled(self):
 		# `.disabled.yaml` is read from the first configured library
 		# It is applied on all libraries in the configuration.
 		disabled = await self.Libraries[0].read('/.disabled.yaml')
+		try:
+			self.Disabled = yaml.safe_load(disabled)
+		except Exception:
+			L.exception("Failed to parse '/.disabled.yaml'")
```
Comment (ateska) [inline asab/library/service.py:138]: `disabled` could be `None`, right?

**Review by ateska — CHANGES_REQUESTED** [review]

File: asab/library/service.py (hunk 1 above)
Comment (mithunbharadwaj) [reply asab/library/service.py:138]: 1. yes, it is the case when .`disabled.yaml `files do not exist.

There is another case where .`disabled.yaml `exists and the file is empty in that I reckon in this scenario

I should set self.Disabled to {}

**Review by ateska — APPROVED** [review]

**Comment by ateska** [comment]
This merge request is incorrect and I close that.

**Comment by ateska** [comment]
See #317

**Pull request closed without merging**